R/search-pv.R

```diff
@@ -96,31 +95,35 @@ request_apply <- function(ex_res, method, query, base_url, arg_list, api_key, ..
   if (req_pages < 1) {
     stop2("No records matched your query...Can't download multiple pages")
   }
-  if (matched_records > 10000) {
```

Comment:
Are we sure that this limitation is going to be dropped permanently? I think I saw something on the docs that said that there *would* be a limitation, which makes me think that the lack of limitation is just a temporary thing.

Reply:
I'm not sure if it's permanent or temporary.  They've been slow to fix reported bugs and haven't announced changes in advance.  I usually notice changes when test cases start failing or they update the OpenAPI spec (I make head requests to see if it's change recently).